ci-operator/config/openshift/ovn-kubernetes/openshift-ovn-kubernetes-release-4.18__periodics.yaml

```diff
@@ -0,0 +1,85 @@
+base_images:
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  hypershift-operator:
+    name: hypershift-operator
+    namespace: hypershift
+    tag: latest
+  ocp_4.17_base-rhel9:
+    name: "4.18"
+    namespace: ocp
+    tag: base-rhel9
+  ocp_builder_rhel-8-golang-1.22-openshift-4.17:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.22-openshift-4.17
+  ocp_builder_rhel-9-golang-1.22-openshift-4.17:
+    name: builder
+    namespace: ocp
+    tag: rhel-9-golang-1.22-openshift-4.17
+  tests-private:
+    name: tests-private
+    namespace: ci
+    tag: latest
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.22
+images:
+- dockerfile_path: Dockerfile.base
+  inputs:
+    ocp_4.17_base-rhel9:
+      as:
+      - registry.ci.openshift.org/ocp/4.17:base-rhel9
+  to: ovn-kubernetes-base
+- dockerfile_path: Dockerfile
+  from: ovn-kubernetes-base
+  inputs:
+    ocp_builder_rhel-8-golang-1.22-openshift-4.17:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.22-openshift-4.17
+    ocp_builder_rhel-9-golang-1.22-openshift-4.17:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.22-openshift-4.17
+  to: ovn-kubernetes
+- dockerfile_path: Dockerfile.microshift
+  from: ovn-kubernetes-base
+  inputs:
+    ocp_builder_rhel-9-golang-1.22-openshift-4.17:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.22-openshift-4.17
+  to: ovn-kubernetes-microshift
```

Comment:
Maybe this has to reference 4.18 ? not sure though

Reply:
Good catch. 

I've updated them for 4.18 periodics. 

Unfortunately, I couldn't find a similar configuration for 4.19. Seems we're the first ones to do that.
I don't even know which images I should be using / which golang version will 4.19 run on.

Reply:
Hm, those fail. 

Looking again in 4.18 presubmits these are also using 4.17:
https://github.com/openshift/release/blob/594a1bdd76bd7435cb216eae16e880f56513e80f/ci-operator/config/openshift/ovn-kubernetes/openshift-ovn-kubernetes-release-4.18.yaml#L31